### PR TITLE
Update presence-admins.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
based on icm work.

### CODE REVIEW

File: Teams/presence-admins.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ appliesto:
 # User presence in Teams
 
 Presence is part of a user's profile in Microsoft Teams (and throughout Microsoft 365 or Office 365). Presence indicates the user's current availability and status to other users. By default, anyone in your organization using Teams can see (in nearly real time) if other users are available online. Presence is updated in real time on the web and desktop versions when you refresh the page on mobile.
-
+ > [!NOTE]
+ > When user presence is changed in graph api, teams client will take few minutes to update the presence, due it is using poll mode.
```
Comment (JohanFreelancer9) [inline Teams/presence-admins.md:28]:
```suggestion
 > When user presence changes in Graph API, the Teams client will take few minutes to update the presence due it using poll mode.
 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RuiPereiraTabares, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@RuiPereiraTabares we wouldn't include details like this in an admin article. The API docs or a support article would be a better option.

**Pull request closed without merging**